src/join/composer.jl

```diff
@@ -233,6 +274,13 @@ function compose_joined_table(joiner::DataFrameJoiner, kind::Symbol, makeunique:
     return res, src_indicator
 end
 
+function _noon_compose_helper(cols, similar_col, cols_i, col, target_nrow,
+                              side_ixs, offset, sideonly_ixs, tocopy)
```

Comment:
Can you add types to the signature? Maybe also add a `!` at the end of the function name.

Reply:
OK - fixed